htdocs/core/class/commonobject.class.php

```diff
@@ -49,6 +51,7 @@
 abstract class CommonObject
 {
 	use DolDeprecationHandler;
+	use CommonPeople;
```

Comment:
ok, i discover trait in php and i don't now this notion, i need the getBannerAddress method to show phone and mail in RecruitmentCandidature card, i don't know how to correct it also, maybe i need to implement this method to RecruitmentCandidature.class ? can you tell me please what do you want ? thanks

Reply:
> ok, i discover trait in php and i don't now this notion, i need the getBannerAddress method to show phone and mail in RecruitmentCandidature card, i don't know how to correct it also, maybe i need to implement this method to RecruitmentCandidature.class ? can you tell me please what do you want ? thanks

Can't help on gow to fix the bug because i don't know what is bug exactly.

You must first describe how to reproduce the bug.

Reply:
in dolibarr v 20 in the banner of recruitementcandidature, the phone and the mail of the candidate is not yet display because reccuitementcandidatuire object doon't have acces to method getbanneradress, before this method was in commonobject, reccuitementcandidatuire extends commonobject, and now getbanneradress is on the trait commonpoeple and not in commonobject, you can't display phone and mail in banner of  recruitementcandidature.
 here with trait include : 
![image](https://github.com/user-attachments/assets/a9ae9a37-02c8-4f46-8531-18f272606787)

Without trait : 
![image](https://github.com/user-attachments/assets/3437429f-6a45-4caf-a7ae-ce76021aadcd)

Reply:
thanks for your returns

Reply:
I see now.
I pushed this fix: https://github.com/Dolibarr/dolibarr/commit/63073f7e712fcb94a379991762321f91557be270
that should be better.